charmtools/build/__init__.py

```diff
@@ -119,6 +115,7 @@ class Builder(object):
     Handle the processing of overrides, implements the policy of BuildConfig
     """
     PHASES = ['lint', 'read', 'call', 'sign', 'build']
+    HOOK_TEMPLATE_FILE = path('hooks/hook.template')
```

Comment:
It seems odd that we define this several times. Here, L12 & L202 of tactics.py, etc